src/github.js

```diff
@@ -28,6 +29,15 @@ Plan: {{ changes.resources.create }} to add, {{ changes.resources.update }} to c
 {{ plan|safe }}
 \`\`\`
 
+</details>
+
+<details>
```

Comment:
Could we hide this `<details>` block if `results.conftest.output` is empty?

Reply:
Yeah